README.md

```diff
@@ -15,12 +15,19 @@ This template includes setup for:
 
 The Next.js application is dockerized. Take a look at `./app/Dockerfile` to see how it works.
 
-A very simple `docker-compose.yml` has been included to support local development and deployment. Take a look at `./docker-compose.yml` for more information.
-
-How to run:
+A `docker-compose.yml` has been included to support local development and deployment. Take a look at `./docker-compose.yml` for more information.
 
 1. In your terminal, `cd` to this repo.
 2. Make sure you have [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed & running.
 3. Run `docker-compose up -d --build` to build the image and start the container.
 4. Navigate to `localhost:3000` in your browser to view the application.
-5. Run `docker-compose down` when you are done to delete the container.
+5. Navigate to `localhost:6006` in your browser to view storybook.
```

Comment:
I ran through the instructions above but get this error now on both Next and Storybook URLs

<img width="1057" alt="CleanShot 2022-06-27 at 10 27 20@2x" src="https://user-images.githubusercontent.com/371943/176000454-3857603f-9d14-482d-a0c5-c9c2dae3fada.png">